deepchem/models/torch_models/unet.py

```diff
@@ -0,0 +1,124 @@
+from deepchem.models.torch_models.modular import TorchModel
+from deepchem.models.losses import BinaryCrossEntropy
+import torch
+import torch.nn as nn
+
+
+class UNet(nn.Module):
```

Comment:
please add docs and example to this

Reply:
Done

Reply:
Also add this to the docs/ as a new model. And add it to the table of models